src/Prefs.cpp

```diff
@@ -63,6 +63,9 @@
 #include "MemoryX.h"
 #include <memory>
 
+BoolSetting DefaultUpdatesCheckingFlag{
```

Comment:
This is somewhat confusing. Now we have a single Settings object defined in Prefs.cpp unlike other similar objects for a purpose of fixing a very specific edge case. We at least should open an issue to clean that up in the master branch after the release.

Reply:
I explained in commit comments that the setting object moved to avoid adding a big dependency cycle in the source code.  A better fix, avoiding the cycle and the move of the definition, would have been more complicated.

As lib-preferences is one of the nearer tasks for 3.1, I could revisit this then.